protocol/protocol.proto

```diff
@@ -994,12 +996,20 @@ message RemoveStreamFromSyncResponse {}
 
 // ModifySyncRequest adds or removes streams from an existing sync session.
 message ModifySyncRequest {
+  message Backfill {
+    // backfill_streams is the list of sync cookies to backfill
+    repeated SyncCookie streams = 1;
+    string sync_id = 2;
+  }
```

Comment:
How is this different than the add streams list and why would it need a unique syncid?